src/lib/theme.css

```
@@ -506,6 +506,90 @@
   --pill-cursor: inherit;
 }
 
+/* ===== RelativeTime ===== */
+.terminal-time {
+  --relative-time-font-size: var(--text-xs);
+  --relative-time-color: var(--text-tertiary);
+  --relative-time-font-weight: 400;
+  white-space: nowrap;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)